server/di/di.go

```diff
@@ -46,6 +48,16 @@ func NewDIContainer(client clientset.Interface, restclientCfg *restclient.Config
 
 	c.priorityClassService = priorityclass.NewPriorityClassService(client)
 	c.exportService = export.NewExportService(client, c.podService, c.nodeService, c.pvService, c.pvcService, c.storageClassService, c.priorityClassService, c.schedulerService)
+
+	deleteServices := map[string]reset.DeleteService{
+		"node": c.nodeService,
+		"pod":  c.podService,
+		"pv":   c.pvService,
+		"pvc":  c.pvcService,
+		"sc":   c.storageClassService,
+		"pc":   c.priorityClassService,
```

Comment:
Could you change it to use the full names of resources for better loggings? (like "Persistent volume")